### PR TITLE
feat: Qdrant support

### PR DESCRIPTION
**Pull request by Anush008**
## Description

This PR adds support for Qdrant to used as a vector database when setting up projects with `create-llama`.

## Summary by CodeRabbit

- **New Features**
    - Added support for Qdrant as a vector database option, including environment variable configurations for server details.
    - Introduced Python and TypeScript functionalities for creating new indexes, uploading documents, and managing connections to Qdrant collections.
    - Enhanced the selection of vector database options to include Qdrant, based on the chosen framework.

### CODE REVIEW

File: templates/components/vectordbs/python/qdrant/generate.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        storage_context=storage_context,
+        show_progress=True,  # this will show you a progress bar as the embeddings are created
+    )
+    logger.info(
+        f"Successfully uploaded documents to the {os.getenv('QDRANT_COLLECTION')} collection."
+    )
+
+
+if __name__ == "__main__":
+    init_settings()
```
Comment (marcusschiesser) [inline templates/components/vectordbs/python/qdrant/generate.py:38]: move `init_settings()` to `generate_datasource` (sorry, just changed this pattern today)

**Review by marcusschiesser — COMMENTED** [review]
@Anush008 thanks that looks great! Please have a look at my comment

File: templates/components/vectordbs/python/qdrant/generate.py (hunk 1 above)
Comment (Anush008) [reply templates/components/vectordbs/python/qdrant/generate.py]: https://github.com/run-llama/create-llama/pull/42/commits/435072e36025540e223b56e96a7e08f4dbf2969d

**Review by marcusschiesser — APPROVED** [review]
Thanks @Anush008

**Pull request merged**